### PR TITLE
Script to set nym rev

### PR DESCRIPTION
**Pull request by octol**
- Add script to list nym git dependencies
- Add script to set the nym git dependencies to either latest or a specific rev, depending on input

- - -
This change is [Reviewable](https://reviewable.io/reviews/nymtech/nym-vpn-client/1750)

### CODE REVIEW

File: nym-vpn-core/Makefile [hunk 1]
```diff
@@ -24,9 +27,10 @@ build-mac: ## Build the Rust workspace suitable for running the daemon
 build-win-cross:
 	cargo build --target x86_64-pc-windows-gnu --release
 
-deb: build-deb-vpn-cli build-deb-vpnd build-deb-vpnc ## Build debian packages
+# -----------------------------------------------------------------------------
+#  Linting targets
+# -----------------------------------------------------------------------------
 
-# Linting targets
```
Comment (pronebird) [inline nym-vpn-core/Makefile:29]: I think we had a script parsing these as a part of help for make. Is it still functional?

**Review by pronebird — CHANGES_REQUESTED** [review]

File: nym-vpn-core/Makefile (hunk 1 above)
Comment (octol) [reply nym-vpn-core/Makefile:29]: Ah you're right, accidentally removed the one for `deb:`. Restored now

**Review by pronebird — APPROVED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/nymtech/nym-vpn-client/1750)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @octol)

**Review by pronebird — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/nymtech/nym-vpn-client/1750#-:-ODfqEHY0A6VUIycCuep:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @octol)

**Pull request merged**